CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
+## [1.7.2]
```

Comment:
should be [X.X.X] - version depends on features added, it may have another features with this one.

Reply:
done